.gitignore

```diff
@@ -4,7 +4,6 @@
 /.cache
 /db
 /dbmate
-/testdata/db/schema.sql
```

Comment:
The schema dumped as a result of test runs should not be commited to the repo.  I think this commit needs to be reverted.

Reply:
Since I originally included the schema file in the repository code this was required to add into repository